en/hosting_guide/configuration.rst

```diff
@@ -12,42 +242,56 @@ Cron jobs typically automate system maintenance.
 Cron jobs on Open Orchestra
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Open Orchestra has 3 cron jobs created with the provisioning:
+Open Orchestra has 4 cron jobs created with the provisioning:
+
+Site maps
+~~~~~~~~~
+
+Generate sitemap files for every sites, further information about `sitemap`_
```

Comment:
i would put either :

 - further information about it : 
 - more information available in the `sitemap`_ documentation